miso-web/src/main/webapp/pages/editLibrary.jsp

```diff
@@ -400,16 +400,16 @@
 <c:when test="${library.id == 0}">
   <tr>
     <td class="h">Tissue Origin:</td>
-    <td>${library.sample.sampleAdditionalInfo.tissueOrigin.alias}</td>
+    <td>${library.sample.tissueOrigin.alias}</td>
```

Comment:
library.sample is not going to be a tissue, so won't have tissueOrigin or tissueType